hack/versions/pkg/schema/check.go

```diff
@@ -90,39 +88,35 @@ func RunSchemaCheckOnChangedFiles() error {
 			continue
 		}
 
-		logrus.Infof("structural changes in latest config, checking on Github if latest is released...")
+		logrus.Warnf("Detected changes to the latest config. Checking on Github if it's released...")
 		latestVersion, isReleased := GetLatestVersion()
 		if !isReleased {
-			color.Green.Fprintf(os.Stdout, "%s is unreleased, it is safe to change it.\n", latestVersion)
+			logrus.Infof("Schema %q is not yet released. Changes are ok.", latestVersion)
 			continue
 		}
-		color.Red.Fprintf(os.Stdout, "%s is released, it should NOT be changed!\n", latestVersion)
+
+		logrus.Errorf("Schema %q is already released. Changing it is not allowed.", latestVersion)
+
+		fmt.Printf("\nWhat should I do?\n-----------------\n")
+		fmt.Printf(" + Please first create a new PR, with a new version, using 'hack/new_version.sh' script.\n")
+		fmt.Printf(" + If you see this error and it seems incorrect, maybe you have an out-of-date %q local branch.\n", baseRef)
+		fmt.Printf(" + If this retroactive change is required and harmless to the users, you'll need Admin rights to merge this PR.\n")
```

Comment:
I think this is much better!  But from the perspective of a new contributor, how would I know if this error seems incorrect?

Here's my suggested tweaks:

```suggestion
		fmt.Printf(" + If this retroactive change is required and is harmless to users, indicate on your PR.\n")
		fmt.Printf(" + Check if a new unreleased version has been created:\n")
		fmt.Printf("     - Ensure that your branch is up-to-date with the %q branch.\n", baseRef)
		fmt.Printf("     - Check for a pending PR to create a new version.\n")
		fmt.Printf(" + Create a separate PR with just the result of running the 'hack/new_version.sh' script.\n")
```